interface/billing/billing_process.php

```diff
@@ -244,7 +244,7 @@ function process_form($ar)
                     }
                 } elseif (isset($ar['bn_process_hcfa'])) {
                     $log = '';
-                    $lines = gen_hcfa_1500($patient_id, $encounter, $log);
+                    $lines = HCFA_1500::gen_hcfa_1500($patient_id, $encounter, $log);
```

Comment:
I think in this case you may need to initiate this class:
$hfca = new HCFA_1500();
$lines = $hfca->gen_hcfa_1500($patient_id, $encounter, $log);